frontend/src/components/MealsList.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import MealItem from "./MealItem";
+
+function MealsList() {
+  const [meals, setMeals] = useState();
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get("http://127.0.0.1:5000/api/meals");
+        if (response.data !== null) {
+          console.log(response.data);
```

Comment:
as this is an assignment its fine, but keep in mind it's not a good practice to leaving unnecessary console logs in production code :)